packages/dumb/src/components/stats/insulin/insulin-stat.tsx

```diff
@@ -108,10 +111,11 @@ const InsulinStat: FunctionComponent<TotalInsulinStatProps> = (props) => {
           className={styles.boldValue}
         >
           <span className={styles.titleTotal}>
-            {totalInsulin}
-          </span>
-          <span className={styles.titleSuffix}>
-            U
+            <Chip
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏I think the "Pen bolus" and "Manual bolus" values were not supposed to be displayed if there were none

>if there are manual boluses and pen boluses display them, if not do not display the text : 
>- manual boluses  quantity and percentage
>- pen boluses quantity and percentage

And according to my test with Sandbox data, the "Pen bolus" line seems to be displayed even when there is no data

![image](https://github.com/user-attachments/assets/8edd6ee7-3bb4-4765-a0f8-65ebfd3cc252)